hw7_functional/functional.py

```diff
@@ -0,0 +1,29 @@
+def sequential_map(*args):
+    container = args[-1]
+    functions = args[:-1]
+    for fun in functions:
+        answer = list(map(fun, container))
+    return answer
```

Comment:
Превратить элемент в список можно только в самом конце, результат map можно передавать в другие map'ы
```suggestion
        answer = map(fun, container)
    return list(answer)
```